packages/ui-components/src/components/Modal/Modal.test.tsx

```diff
@@ -382,7 +382,21 @@ describe("Modal", () => {
     expect(cancelButton).toBeInTheDocument()
     expect(cancelButton).toHaveAttribute("disabled")
 
+    expect(screen.getByRole("button", { name: "close" })).toBeDisabled()
+
     await waitFor(() => userEvent.click(cancelButton))
     expect(mockOnCancel).not.toHaveBeenCalled()
   })
+
+  test("renders a modal with no disabled close button if disableCloseButton is true", async () => {
```

Comment:
Shouldn't that read "renders a modal with **_a_** disabled close button if disableCloseButton is true"?